ml_tools/eolearn/tests/test_sampling.py

```diff
@@ -215,77 +244,107 @@ def test_fraction_sampling(
     sampled_uniques, sampled_counts = np.unique(eopatch.data["NDVI_SAMPLED"], return_counts=True)
     masked = eopatch.mask_timeless["sampling_mask"].squeeze(axis=2) == 1
     masked_uniques, masked_counts = np.unique(eopatch.data["NDVI"][:, masked], return_counts=True)
-    assert_array_equal(sampled_uniques, masked_uniques, err_msg="Sampling mask not correctly describing sampled points")
-    assert_array_equal(sampled_counts, masked_counts, err_msg="Sampling mask not correctly describing sampled points")
+    assert_array_equal(sampled_uniques, masked_uniques)
+    assert_array_equal(sampled_counts, masked_counts)
+
+
+@pytest.mark.parametrize("seed", range(3))
+@pytest.mark.parametrize("fraction_task", [[0.2, None], [0.4, [0, 1]]], indirect=True)
+def test_fraction_sampling(fraction_task: FractionSamplingTask, seed: int, test_eopatch: EOPatch) -> None:
```

Comment:
change the name of this and `test_fraction_sampling_fine_tuned` to something like `test_fraction_sampling_input_fraction` and `test_fraction_sampling_input_dict`

Reply:
done